crates/intrinsic-test/src/common/gen_rust.rs

```diff
@@ -130,7 +130,7 @@ pub fn setup_rust_file_paths(identifiers: &Vec<String>) -> BTreeMap<&String, Str
     identifiers
         .par_iter()
         .map(|identifier| {
-            let rust_dir = format!(r#"rust_programs/{}"#, identifier);
+            let rust_dir = format!(r#"rust_programs/{identifier}"#);
```

Comment:
```suggestion
            let rust_dir = format!("rust_programs/{identifier}");
```

This doesn't need to be a raw string.